.bazelversion

```diff
@@ -1 +0,0 @@
-3.3.1
```

Comment:
Should this be part of the diff?

Reply:
oops, reverting it :)
Was testing it on several bazel versions